MANUAL.html

```diff
@@ -1171,8 +1172,10 @@ <h3 id="copy">Copy</h3>
 <h3 id="move">Move</h3>
 <p>Used when moving/renaming an object on the same remote. This is known as a server side move of a file. This is used in <code>rclone move</code> if the server doesn't support <code>DirMove</code>.</p>
 <p>If the server isn't capable of <code>Move</code> then rclone simulates it with <code>Copy</code> then delete. If the server doesn't support <code>Copy</code> then rclone will download the file and re-upload it.</p>
+<h3 id="dircopy">DirCopy</h3>
```

Comment:
Note that you only need to edit `docs/content/overview.md` - the other files are auto generated when I do a release.  See [the contribution manual](https://github.com/ncw/rclone/blob/master/CONTRIBUTING.md#writing-documentation)